crates/nu-command/tests/commands/default.rs

```diff
@@ -32,3 +32,15 @@ fn default_after_empty_filter() {
 
     assert_eq!(actual.out, "d");
 }
+
+#[test]
+fn keeps_nulls_in_lists() {
+    let actual = nu!(r#"[null, 2, 3] | default []"#);
+    assert_eq!(actual.out, "[null, 1, 2, 3]");
```

Comment:
This doesn't look right

Reply:
Our tests are slightly weird, we test by calling the whole binary which will `table` format the result. So, this I think should do it:
```suggestion
    let actual = nu!(r#"[null, 2, 3] | default [] | to nuon"#);
    assert_eq!(actual.out, "[null, 1, 2, 3]");
```

Reply:
I added `to json -r` before I saw your comment.

Reply:
All good, that works!